orbstation/code/antagonists/blood_brothers.dm

```diff
@@ -31,6 +31,36 @@
 /datum/objective/steal/brothers
 	/// Where the objective is to be delivered to
 	var/delivery_site
+	/// Don't try to deliver these items
+	var/static/list/brothers_blacklist = list(
+		/datum/objective_item/steal/blackbox,
+		/datum/objective_item/steal/blueprints,
+		/datum/objective_item/steal/documents,
+		/datum/objective_item/steal/functionalai,
+		/datum/objective_item/steal/hdd_extraction,
+		/datum/objective_item/steal/nukedisc,
+		/datum/objective_item/steal/nuke_core,
+		/datum/objective_item/steal/supermatter,
+	)
+
+/datum/objective/steal/brothers/find_target(dupe_search_range, list/blacklist)
+	var/list/datum/mind/owners = get_owners()
+	if(!dupe_search_range)
+		dupe_search_range = get_owners()
+	var/approved_targets = list()
+	for(var/datum/objective_item/possible_item in GLOB.possible_items)
+		if(!possible_item.valid_objective_for(owners, require_owner = FALSE))
```

Comment:
should this be equals true?

Reply:
For future reference: No its optional because the way we made it work upstream is that `require_owner` is set to TRUE regardless of what you pass in.
But it's good to be clear about what we want.